addons/schedule/common/src/main/java/org/commonjava/indy/schedule/ScheduleDBUtil.java

```diff
@@ -0,0 +1,36 @@
+package org.commonjava.indy.schedule;
+
+import org.commonjava.indy.schedule.conf.ScheduleDBConfig;
+
+public class ScheduleDBUtil
+{
+
+    public static final String TABLE_SCHEDULE = "schedule";
+
+    public static String getSchemaCreateKeyspace( ScheduleDBConfig config, String keyspace )
+    {
+        return "CREATE KEYSPACE IF NOT EXISTS " + keyspace
+                        + " WITH REPLICATION = {'class':'SimpleStrategy', 'replication_factor':"
+                        + config.getReplicationFactor() + "};";
+    }
+
+    public static String getSchemaCreateTableSchedule( String keyspace )
+    {
+        return "CREATE TABLE IF NOT EXISTS " + keyspace + "." + TABLE_SCHEDULE + " ("
+                        + "jobtype varchar,"
```

Comment:
What is the jobtype supposed to hold?

Reply:
The initial types are the following two: CONTENT and Disabled-Timeout.  
addons/schedule/model-java/src/main/java/org/commonjava/indy/schedule/datastax/JobType.java

Reply:
why not add it to primary key? this would save the index.

Reply:
or you use/index it to query across multiple stores?

Reply:
Yeah, the case is we need to query all the schedules that is type of "Disabled-Timeout",  no matter with the store.

Reply:
Ok. Just be aware that cross partion query is going to be slow even with index. If those queries are very often, this would be problem.